lib/email_reply_parser.rb

```diff
@@ -188,7 +194,7 @@ def scan_line(line)
     #
     # Returns true if the line is a valid header, or false.
     def quote_header?(line)
-      line =~ /^:etorw.*nO$/
+      line =~ /^:etorw.*nO$/ || line =~ /^.*:(morF|tneS|oT|tcejbuS)$/
```

Comment:
I'm not familiar with this codebase, just trying to move this along. Why is this matching what appear to be reversed strings? It's consistent with the previous regular expression, but I'm having a hard time understanding why this might be.

Reply:
@derekprior Honestly, I don't know. I was also having a hard time understanding why this was the case, figured there is probably a reason so just stuck to it.